tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/Test_NeighborPackagedData.cpp

```diff
@@ -0,0 +1,304 @@
+      typename variables_tag::type{dg_mesh.number_of_grid_points()},
+      face_centered_gr_tags(subcell_mesh, time, coordinate_map, soln),
+      neighbor_data, 1.0, evolution::dg::Tags::MortarData<3>::type{},
+      std::optional<tnsr::I<DataVector, 3, Frame::Inertial>>{}, normal_vectors);
+  db::mutate_apply<ConservativeFromPrimitive>(make_not_null(&box));
+
+  std::vector<std::pair<Direction<3>, ElementId<3>>>
+      mortars_to_reconstruct_to{};
+  for (const auto& [direction, neighbors] : element.neighbors()) {
+    mortars_to_reconstruct_to.emplace_back(direction, *neighbors.begin());
+  }
+
+  const auto all_packaged_data =
+      subcell::NeighborPackagedData::apply(box, mortars_to_reconstruct_to);
+
+  // Parse out evolved vars, since those are easiest to check for correctness,
+  // then return absolute difference between analytic and reconstructed values.
+  FixedHashMap<maximum_number_of_neighbors(3),
+               std::pair<Direction<3>, ElementId<3>>,
+               typename variables_tag::type,
+               boost::hash<std::pair<Direction<3>, ElementId<3>>>>
+      evolved_vars_errors{};
+  double max_abs_error = 0.0;
+  for (const auto& [direction_and_id, data] : all_packaged_data) {
+    const auto& direction = direction_and_id.first;
+    using dg_package_field_tags = typename grmhd::ValenciaDivClean::
+        BoundaryCorrections::Hll::dg_package_field_tags;
+    const Mesh<2> face_mesh = dg_mesh.slice_away(direction.dimension());
+    Variables<dg_package_field_tags> packaged_data{
+        face_mesh.number_of_grid_points()};
+    std::copy(data.begin(), data.end(), packaged_data.data());
+
+    auto sliced_vars = data_on_slice(
+        db::get<variables_tag>(box), dg_mesh.extents(), direction.dimension(),
+        direction.side() == Side::Upper
+            ? dg_mesh.extents(direction.dimension()) - 1
+            : 0);
+
+    tmpl::for_each<typename variables_tag::type::tags_list>(
+        [&sliced_vars, &max_abs_error, &packaged_data](auto tag_v) {
+          using tag = tmpl::type_from<decltype(tag_v)>;
+          auto& sliced_tensor = get<tag>(sliced_vars);
+          const auto& packaged_data_tensor = get<tag>(packaged_data);
+          for (size_t tensor_index = 0; tensor_index < sliced_tensor.size();
+               ++tensor_index) {
+            max_abs_error = std::max(
+                max_abs_error, max(abs(sliced_tensor[tensor_index] -
+                                       packaged_data_tensor[tensor_index])));
+          }
+        });
+  }
+  return max_abs_error;
+}
+
+SPECTRE_TEST_CASE(
+    "Unit.Evolution.Systems.ValenciaDivClean.Subcell.NeighborPackagedData",
+    "[Unit][Evolution]") {
+  // This tests sets up a cube [2,3]^3 in a Bondi-Michel spacetime and verifies
+  // that the difference between the reconstructed evolved variables and the
+  // sliced (exact on LGL grid) evolved variables on the interfaces decreases.
+  CHECK(test(4) > test(8));
```

Comment:
As far as I can tell, this is the only check in the test, and it doesn't seem like a terribly strong check of correctness; just that the order 8 version is some improvement. Could you add some check that also involves an absolute or relative tolerance for the `test(4)` and `test(8)` results individually?